src/pages/index.tsx

```diff
@@ -115,8 +116,32 @@ export function Index(props: RouteComponentProps) {
                     </Box>
                   ))}
                 </Box>
-                <Text sx={{lineHeight: '1'}}>{palette.name}</Text>
+                <Text sx={{ lineHeight: '1' }}>{palette.name}</Text>
               </Box>
+              <IconButton
+                  aria-label='Delete Palette'
+                  icon={TrashIcon}
+                  onClick={() => {
+                    send({ type: 'DELETE_PALETTE', paletteId: palette.id })
+                  }}
+                  sx={{
+                    color: readableColor(palette.backgroundColor),
+                    backgroundColor: palette.backgroundColor,
+                    borderColor: mix(
+                      readableColor(palette.backgroundColor),
+                      palette.backgroundColor,
+                      0.75),
+                    position: 'absolute',
+                    right: 3,
+                    bottom: 2,
+                    '&:hover:not([disabled])': {
+                      backgroundColor: mix(
+                        readableColor(palette.backgroundColor),
+                        palette.backgroundColor,
+                        0.85),
+                    }
+                  }}
+                >Delete alette</IconButton>
```

Comment:
This is a typo, but: do you need this inner text _and_ the aria-label? Or just one?